ingestion/src/metadata/ingestion/source/dashboard/superset/db_source.py

```diff
@@ -62,26 +74,32 @@ def prepare(self):
         """
         charts = self.engine.execute(FETCH_ALL_CHARTS)
         for chart in charts:
-            self.all_charts[chart["id"]] = dict(chart)
+            chart_detail = FetchChart(**chart)
+            self.all_charts[chart_detail.id] = chart_detail
+
+    def get_column_list(self, table_name: FetchChart) -> Optional[List[object]]:
```

Comment:
we should not mark the return type as `object` here it should be `FetchColumn`

Reply:
Sure will do